resources/quality/dagoma/dagoma_discoeasy200_pla_standard.inst.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+version = 3
+name = Standard
+definition = dagoma_discoeasy200
+
+[metadata]
+setting_version = 4
+type = quality
+quality_type = normal
```

Comment:
I think `quality_type` normal correspond with `layer_height` 0.1. Shouldn't be `fast` here?

Reply:
Fixed in 139ac5c